settings.gradle

```diff
@@ -1,3 +1,6 @@
-rootProject.name = 'xxx'
```

Comment:
When you rename this you didn't remove the `xxx` directory.

Reply:
Thanks, I have deleted it.